lib/engine/game/g_1893/step/starting_package_forced_auction.rb

```diff
@@ -0,0 +1,192 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+require_relative '../../../step/passable_auction'
+require_relative 'buy_minor'
+
+module Engine
+  module Game
+    module G1893
+      module Step
+        class StartingPackageForcedAuction < Engine::Step::Base
+          include Engine::Step::PassableAuction
+          include BuyMinor
+
+          ACTIONS = %w[bid pass].freeze
+
+          def description
+            'Drafting Auction for Remaining Starting Package Certificates'
+          end
+
+          def help
+            return 'Select a company to auction' unless @auctioning
+
+            "Buy or Pass. If everyone pass price is lowered by #{@game.format_currency(10)} and a new buy/pass " \
+              'opportunity is presented. This continues until someone buys it or when price would go below 50% ' \
+              'of the value in which case the  player that selected the company is force to buy it. ' \
```

Comment:
double space between "the" and "player"

```suggestion
              'of the value in which case the player that selected the company is force to buy it. ' \
```

Reply:
Fixed